### PR TITLE
#72 BillingAddressTrait and ShippingAddressTrait

### LINKED ISSUE #72: Remove code duplications reported by Scrutinizer
Link to report: https://scrutinizer-ci.com/g/xsolve-pl/salesforce-client/issues/master?selectedLabels%5B0%5D=4&orderField=path&order=asc&honorSelectedPaths=0

### PR DESCRIPTION
**Pull request by adrianwaler**
Fixes #72

### CODE REVIEW

File: src/Model/BillingAddressTrait.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+namespace Xsolve\SalesforceClient\Model;
+
+use Xsolve\SalesforceClient\Model\ValueObject\Address;
+use JMS\Serializer\Annotation as JMS;
+
+trait BillingAddressTrait
+{
+    /**
+     * @var string|null
```
Comment (krzysztofc-xsolve) [inline src/Model/BillingAddressTrait.php]: Is this optional for every request?

Comment (adrianwaler) [inline src/Model/BillingAddressTrait.php]: Yes, like everything in both traits.

**Comment by Kryniol** [comment]
Tobias from Scrutinizer wrote

>Hi Paweł, thank you for your report. This seems to be an error in our rating system. Account should be 
> rated B as well or at least remain on C because of the higher total lines. I created an internal issue for it. 
> If you need anything else, just let me know.

Let's postpone merging this one a bit.

**Comment by Kryniol** [comment]
I checked, the issue still persists

**Pull request merged**